src/Sheet.php

```diff
@@ -352,6 +353,10 @@ public function toArray($import, int $startRow = null, $nullValue = null, $calcu
 
             $row = $row->toArray($nullValue, $calculateFormulas, $formatData, $endColumn);
 
+            if ($import instanceof IsRowEmpty && $import->isEmptyWhen($row)) {
```

Comment:
I think we can do it without the concern and do a `method_exists` check just like with prepareForValidation

Reply:
Thanks for your feedback, Done